PocketShop/CustomerViews/CustomerHomeView.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+
+struct CustomerHomeView: View {
+    @StateObject var viewModel = CustomerViewModel()
+
+    var body: some View {
+        NavigationView {
+            VStack {
+                Text("Products")
+                    .font(.title)
+                    .fontWeight(.bold)
+                    .foregroundColor(Color.black)
+
+                ScrollView(.horizontal, showsIndicators: false) {
+                    HStack {
+                        ForEach(viewModel.productSearchResults, id: \.self) { product in
+                            NavigationLink(destination: ProductView(product: product)) {
+                                ProductSummaryView(product: product)
+                            }
+                        }
+                    }
+                }
+                .searchable(text: $viewModel.searchText)
```

Comment:
I'll make a fix to this for my iDevices since I'm on iOS 14.3, and `searchable` is only compatible with > iOS 15